packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx

```diff
@@ -0,0 +1,320 @@
+import React, { useEffect } from 'react'
+import useMeasure from 'react-use-measure'
+
+import { GridItem } from '@/components/LayoutGrid'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { SvgActionChevronB } from '@/components/_icons'
+import { absoluteRoutes } from '@/config/routes'
+import { cVar } from '@/styles'
+import { formatNumberShort } from '@/utils/number'
+import { formatDateTime, formatDurationShort } from '@/utils/time'
+
+import {
+  ButtonGrid,
+  Container,
+  Content,
+  InfoItemContainer,
+  InfoItemContent,
+  JoyTokenIcon,
+  Label,
+  NFTHistoryHeader,
+  NFTOwnerContainer,
+  OwnerAvatar,
+  OwnerHandle,
+  OwnerLabel,
+  SecondaryText,
+  TimerSecondaryText,
+} from './NftWidget.styles'
+
+type Size = 'medium' | 'small'
+
+export type Auction = {
+  status: 'auction'
+  startingPrice: number
+  buyNowPrice?: number
+  topBid?: number
+  isCompleted?: boolean
+  canWithdrawBid?: boolean
+  auctionPlannedEndDate?: Date
+}
+
+export type NftWidgetProps = {
+  ownerHandle?: string
+  ownerAvatarUri?: string
+  isOwner: boolean
+  nftState:
+    | { status: 'iddle'; lastPrice?: number; lastTransactionDate?: Date }
```

Comment:
```suggestion
    | { status: 'idle'; lastPrice?: number; lastTransactionDate?: Date }
```

This will need update in other places as well